Backend/ReleaseProcess.md

```diff
@@ -0,0 +1,46 @@
+Release process 📦
+==================================
+
+Our backend has a micro-service architecture. 
+To help improve the release process we are using [Semantic Release](https://semantic-release.gitbook.io/semantic-release/) in our commits.
+
+To automate finding the next version, we use [go-semrel-gitlab](https://juhani.gitlab.io/go-semrel-gitlab/) in our pipelines.
+
+
+## Merge requests
+
+Every merge request should run a pipeline to build, test and lint those changes, and only when everything is ✅ the merge can be done. 
+
+The stages for the merge requests are:
+
+![Merge request](./media/merge_request.png)
+
+- `BuildMR` - Build or assemble the project
+- `LintMR` - Run lint checks
+- `TestsMR` - Run tests 
+
+
+## After merge
+
+After the merge is merged (🤯), a new set of jobs should run, to effectively release the new build.
```

Comment:
`After the merge, a new set of jobs will run and effectively release the new build.`  no? 🤔 

perhaps we could specify the strategy, no?

Reply:
What do you think about `After the merge, a new pipeline should run and effectively release the new build.` ? 🤔 

Regarding the merge strategy, I was thinking that that should be in another place, since it , probably, will be common to all the projects

Reply:
Ok ok, under development it will execute automatically.